nix/go-ethereum.nix

```
@@ -0,0 +1,60 @@
+{ lib, stdenv, buildGoModule, fetchFromGitHub, libobjc, IOKit }:
+
+let
+  # A list of binaries to put into separate outputs
+  bins = [
+    "geth"
+    "clef"
+  ];
+
+in
+buildGoModule rec {
+  pname = "go-ethereum";
+  version = "1.10.15";
```

shouldn't this be `1.10.19`?
```suggestion
  version = "1.10.19";
```